qa/pom.xml

```diff
@@ -78,6 +79,12 @@
       <version>2.19.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>jakarta.xml.bind</groupId>
+      <artifactId>jakarta.xml.bind-api</artifactId>
+      <version>${version.jakarta-xml-bind-api}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This dependency was transitively pulled in by `spring-boot-starter-data-jpa` and needs to be defined now manually.